app/main.py

```diff
@@ -14,0 +1,12 @@
+from models import Actor
+from managers import ActorManager
+
+if __name__ == "__main__":
+    Actor.objects = ActorManager()
+    Actor.objects.create(first_name="Emma", last_name="Watson")
+    Actor.objects.create(first_name="Daniel", last_name="Radclife")
+    print(Actor.objects.all())
+    Actor.objects.update(2, "Daniel", "Radcliffe")
```

Comment:
The `update` method in `ActorManager` uses 'lt_nm' instead of 'last_name'. This will cause an error because 'lt_nm' is not a valid column name in the database. Ensure the correct column name 'last_name' is used in the `update` method.